data/tokens.json

```diff
@@ -113,6 +113,11 @@
         "decimals": 6,
         "name": "USD Coin",
         "symbol": "USDC"
+      },
+      "0x3426d85D140c85C5ebB6E4D343C5be8e4E001869": {
```

Comment:
can this contract be verified on etherscan ?

Reply:
Sure, here:
https://rinkeby.etherscan.io/address/0x3426d85D140c85C5ebB6E4D343C5be8e4E001869#code